otx/algorithms/action/configs/detection/base/base_detection_dynamic.py

```diff
@@ -0,0 +1,37 @@
+"""Base Action detection mmdeply cfg."""
+
+ir_config = dict(
+    type="onnx",
+    export_params=True,
+    keep_initializers_as_inputs=False,
+    opset_version=11,
+    save_file="end2end.onnx",
+    input_names=["input"],
+    output_names=["bboxes", "labels"],
+    input_shape=None,
+    optimize=True,
```

Comment:
```suggestion
    optimize=False,
```
`optimize` of `mmedploy` messes up NNCF graph. It would be better to be `False`.